src/npc.cpp

```diff
@@ -2316,25 +2316,20 @@ void npc::process_turn()
     // TODO: Make NPCs leave the player if there's a path out of map and player is sleeping/unseen/etc.
 }
 
+const std::string need_id( npc_need need )
```

Comment:
Why do many of your functions return a const value? It's going to be *copied* into the variable provided by the caller anyway.